README.md

```diff
@@ -2,62 +2,107 @@
 
 This is a CosmWasm smart contract to manage Curve Leverage Lending Bot smart contract on EVM chain written in Vyper.
 
-Users can create a crvUSD loan by deposit their token into a Vyper smart contract with leverage option on EVM chain.
+Users can create a Curve Leverage Lending Bot by depositing their tokens into a Vyper smart contract with options for leverage on the EVM chain.
 
-A scheduler or script fetch events from the Vyper smart contract and run `repay_bot` function with the parameters via Compass-EVM on high risk or expiration.
+A scheduler or script fetches events from the Vyper smart contract and runs `repay_bot` function with the parameters via Compass-EVM on high risk or expiration.
 
-And then, the Vyper smart contract will repay the bot.
+The Vyper smart contract will then repay the bot.
 
 ## ExecuteMsg
 
 ### RepayBot
 
 Run `repay_bot` function on Vyper smart contract.
 
-| Key                        | Type           | Description                     |
-|----------------------------|----------------|---------------------------------|
-| bot_info                   | Vec\<BotInfo\> | Array of data to add collateral |
+| Key                        | Type            | Description                     |
+|----------------------------|-----------------|---------------------------------|
+| bots                       | DynArray[address, MAX_SIZE] | Array of bot addresses |
+| callbackers                | DynArray[address, MAX_SIZE] | Array of callbacker addresses |
+| callback_args              | DynArray[DynArray[uint256,5], MAX_SIZE] | Nested array of callback arguments for each bot |
+
+### CreateBot
```

Comment:
https://github.com/VolumeFi/curve-leverage-lending-bot-cw/blob/1e093a5eb682ce8cd0347becddbdb31827e3b7d6/src/msg.rs#L21
This function has 4 parameters only.